suricata.yaml.in

```diff
@@ -1001,6 +1001,13 @@ asn1-max-frames: 256
 # umask passed on by the shell.
 #umask: 022
 
+# Global memcap
+# This value represents the maximum amount of memory that can be used
+# to allocate all the memcap values.
+# The sum of all of them (stream.memcap, flow.memcap, etc.) cannot be
+# greater than the global memcap.
+#global-memcap: 128mb
```

Comment:
Would a higher value make more sense, especially since with the default suricata.yaml  we would have it higher already?

Reply:
Agreed